openapi/connector_mgmt.yaml

```diff
@@ -957,17 +1299,16 @@ components:
       required:
         - name
         - connector_type_id
-        - deployment_location
         - desired_state
       properties:
         name:
           type: string
         connector_type_id:
           type: string
+        namespace_id:
```

Comment:
This should be listed among the required fields

Reply:
That'll take me another day or so to fix all the tests that'll break in connector-api.feature that don't use a namespace_id at this point. Would you like me to do that on this branch or on main?

Reply:
Yes, will do that before going to staging.

Reply:
I can also just change the API for now without requiring it in the implementation which will break a whole feature. We don't validate the incoming JSON against the API schema.